gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -707,11 +819,17 @@ def main(args):
     v4_genome_release_resources = get_v4_genomes_release_resources(
         test=test, overwrite=overwrite
     )
-    meta_ht = v4_genome_release_resources.meta_ht.ht()
-    dense_mt = v4_genome_release_resources.dense_mt.mt()
+    v3_hgdp_tgp_meta_ht = v4_genome_release_resources.meta_ht.ht()
+    v3_hgdp_tgp_dense_mt = v4_genome_release_resources.dense_mt.mt()
+    v3_hgdp_tgp_sites_ht = v4_genome_release_resources.sites_ht.ht()
```

Comment:
I think the name here is again a bit confusing, can we use `v3_sites_ht` or `v3_release_sites_ht` instead?

Reply:
Oops, yup, I didn't mean to change that one, thanks for catching